translation of "portal-how-to-enable-webhook-notification.md" (#3847)
ctripcorp/apollo: docs/en/development/portal-how-to-enable-webhook-notification.md

> Apollo has supported  **webhook** since 1.8.0 edition, which is triggered to send you a message when your configuration is released.

Reviewer: ```suggestion
Starting with version 1.8.0, Apollo has added webhook support, which will be triggered to send you a message when your configuration is released.
```

Author: Thanks for your suggestion. I checked carefully and corrected  accordingly.

> ## how to enable webhook

Reviewer 2: ```suggestion
## How to enable webhook
```

> The configuration items are stored in the table named ApolloPortalDB.ServerConfig. "admin—system parameter" page is an alternative to configuration. All the configurations will be valid in one minutes.

Reviewer 3: ```suggestion
> The configuration items are stored in the table named ApolloPortalDB.ServerConfig. `admin—system parameter` page is an alternative to configuration. All the configurations will be valid in one minute.
```

> > The configuration items are stored in the table named ApolloPortalDB.ServerConfig. `admin—system parameter` page is an alternative to configuration. All the configurations will be valid in one minute.

Reviewer: ```suggestion
> The configuration items are stored in the table named ApolloPortalDB.ServerConfig. You may also config them in `Admin Tools - System Configuration` page. The configuration changes will take effect within one minute.
```

1. webhook.supported.envs

> Open the environments lists. Multiple envs are split by comma, like

Reviewer: ```suggestion
The environments lists to enabled webhook support. Multiple environments should be separated by commas, e.g.
```

```
DEV, FAT, UAT, PRO
```

2. config.release.webhook.service.url

> Config the url that receives HTTP post request sent by webhook for notifying. Multiple urls should be split by comma, like

Reviewer: ```suggestion
Config the url that receives HTTP post request sent by webhook for notifying. Multiple urls should be separated by commas, e.g.
```

```
http://www.xxx.com/webhook1,http://www.xxx.com/webhook2
```

> ## how to use

Reviewer 2: ```suggestion
## How to use
```

1. URL params

> param name    | param annotation

Reviewer 3: Use `parameter` instead of `param`?
